releases/XCFramework/libzcashlc.xcframework/ios-arm64/libzcashlc.framework/Headers/zcashlc.h

```diff
@@ -396,6 +462,20 @@ struct FFIBinaryKey *zcashlc_create_account(const uint8_t *db_data,
                                             int64_t recover_until,
                                             uint32_t network_id);
 
+/**
+ * Checks whether the given seed is relevant to any of the accounts in the wallet.
+ *
+ * Returns:
+ * - `1` for `Ok(true)`.
+ * - `0` for `Ok(false)`.
```

Comment:
Non-blocking: Ideally, documentation at this level would not depend on knowing the semantics of the corresponding Rust function (which isn't linked here).